openfisca_us/parameters/gov/states/md/tax/income/single.yaml

```
@@ -0,0 +1,45 @@
+description: MD State income tax rate for single filers
```

Rename `single_separate.yaml`
```suggestion
description: MD State income tax rate for single and married filing separately filers
```